docs/source/deployment/azure_container_instance.rst

```diff
@@ -171,14 +199,14 @@ Create and configure Azure ACR (Azure Container Registry)
 
 .. code-block:: bash
 
-    > az acr login --name bentomlirisclassifier
+    az acr login --name bentomlirisclassifier
 
     Login Succeeded
 
 
 .. code-block:: bash
 
-    > az acr show --name BentoMLIrisClassifier --query loginServer --output table
```

Comment:
I think it's actually better to keep ">" or "$" here

It looks really bad now - it's the same format as the stdout result below now